dashboard/app/models/sections/section.rb

```diff
@@ -416,6 +416,8 @@ def concise_summarize
         sectionInstructors: serialized_section_instructors,
         sync_enabled: Policies::Lti.roster_sync_enabled?(teacher),
         ai_tutor_enabled: ai_tutor_enabled,
+        version_year: unit_group ? unit_group&.version_year : script&.version_year,
+        has_calendar: script ? script.show_calendar : false,
```

Comment:
As we discussed earlier, it is probably a better approach to make sure that we are always going to load the calendar page. There are a few options:
* Add a new flag or modify existing flags to only show the empty state if we have loaded the calendar
* Move the empty state into the calendar component so that loading happens before we determine empty states
I think the first approach is better for right now, but then we can refactor the ElementOrEmptyPage to actually make it better.